### PR TITLE
fix: Add missing image registry for eventbus controller

### PR DESCRIPTION
**Pull request by terrytangyuan**
Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-events/templates/eventbus-controller/deployment.yaml [hunk 1]
```diff
@@ -45,9 +45,9 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: NATS_STREAMING_IMAGE
-              value: {{ .Values.eventbusController.natsStreamingImage }}
+              value: "{{ .Values.registry }}/{{ .Values.eventbusController.natsStreamingImage }}"
             - name: NATS_METRICS_EXPORTER_IMAGE
-              value: {{ .Values.eventbusController.natsMetricsExporterImage }}
+              value: "{{ .Values.registry }}/{{ .Values.eventbusController.natsMetricsExporterImage }}"
```
Comment (terrytangyuan) [inline charts/argo-events/templates/eventbus-controller/deployment.yaml:50]: Looks like these images are not on quay.io yet. Should I create a separate parameter, e.g. `natsRegistry` for that? The motivation for this change is that we have users hitting the DockerHub rate limit when pulling these images.

Comment (mkilchhofer) [reply charts/argo-events/templates/eventbus-controller/deployment.yaml:50]: Hmmm. I think we need at least a dedicated key to for the registry of the NATS images as these are external dependencies hosted on a different registry by default.

Maybe we could also leave it as-is and place a reminder into the README as overriding is already possible without this change.

Comment (terrytangyuan) [reply charts/argo-events/templates/eventbus-controller/deployment.yaml:50]: > overriding is already possible without this change

Could you elaborate?

**Comment by terrytangyuan** [comment]
Closing this as the feedback loop is very long and not worth the efforts.

**Pull request closed without merging**